site/dat/docs/Nose.md

```diff
@@ -208,7 +209,23 @@ scenario:
       - storeString(${my_title} love my ${my_text} with ${my_value}): final_text
 ```
 
+### Inline modification
 
+There is low-level way to change generated python file. 
+You can use inline blocks to insert/replace lines of code.
+```yaml
+scenario:
+  sc1:
+    requests:
+    - http://blazedemo.com
+    inline-blocks:
+    - after: 10   # start file if omited
+      before: 20  # `after` + 1 by default. Used for removing lines from `after` + 1 to `before` - 1
+      text: "first line()\nsecond(line)"  # required field
+      ident: auto   # default value, identation based on previous (`after`) line
```

Comment:
`s/identation/indentation`

Reply:
Maybe we need a more concrete example? I'm not sure which, but maybe injecting something like printing browser javascript logs into STDOUT will be a nice example.
```python
for entry in driver.get_log('browser'):
    print(entry)
```